### PR TITLE
Presentation list update

### PR DESCRIPTION
**Pull request by sgnoohc**

### CODE REVIEW

File: _data/people/sgnoohc.yml [hunk 1]
```diff
@@ -8,6 +8,13 @@ shortname: sgnoohc
 title: Tracking Algorithms Collaborator
 website: https://philip.ucsd.edu
 presentations:
+- title: Line segment tracking for Phase-II Tracking
+  date: 2021-10-18
+  url: https://indico.cern.ch/event/1087315/contributions/4571855/attachments/2329818/3969872/PhilipChang20211018_LST_TrackPOG_v5.pdf
```
Comment (henryiii) [inline _data/people/sgnoohc.yml:13]: Could you like to just the overall contribution, and not the full URL? Indico's full URLs to attachments are not stable.

```suggestion
  url: https://indico.cern.ch/event/1087315/contributions/4571855
```

Comment (sgnoohc) [reply _data/people/sgnoohc.yml:13]: Good point, and I thought about this as well.
But given that there is a `meetingurl` below, which would take you to the meeting indico page, I think this is OK.
(On the website, both links are rendered to be shown next to each other.)

**Comment by davidlange6** [comment]
Which sort of indico urls are not stable (or less stable than other indico urls)?

File: _data/people/sgnoohc.yml (hunk 1 above)
Comment (henryiii) [reply _data/people/sgnoohc.yml:13]: It highlights the contribution, at least, and scrolls to it if needed.

**Comment by henryiii** [comment]
Attachement URLS. Minor changes can invalidate attachment URLs. Links to the meeting and the contribution are much more stable. I've had to fix dead attachment links a few times here, finally decided to just always strip off the attachment part.

File: _data/people/sgnoohc.yml (hunk 1 above)
Comment (henryiii) [reply _data/people/sgnoohc.yml:13]: And it's consistent with what we link to on the hundreds of other presentations.

Comment (sgnoohc) [reply _data/people/sgnoohc.yml:13]: > It highlights the contribution, at least, and scrolls to it if needed.

Ah that's one way to solve it to be more stable and lead to the contribution directly. Yes. I will try that.

**Comment by davidlange6** [comment]
Thanks - I didn't know that. (I knew they change if someone removes/reuploads a pdf but that should not be happening after meetings happen)

**Comment by rct225** [comment]
@henryiii @davidlange6 @sgnoohc Can this update be merged?

**Pull request merged**